src/EventListener/AngularHtmlListener.php

```diff
@@ -51,7 +51,7 @@ public function onPostTranspile(AssetEvent $event): void
         $content = preg_replace_callback(
             '/templateUrl\s*:(\s*[\'"`](.*?)[\'"`]\s*)/m',
             function ($match) use ($file, $reader) {
-                return 'template: ' . json_encode($this->getCompiledAssetFor($match[2], $file, $reader));
+                return 'template: ' . json_encode($this->getCompiledAssetFor('./' . $match[2], $file, $reader));
```

Comment:
what if these are absolute paths? Or is that not possible?

Reply:
technically it's possible, but in general it doesn't happen, but I can add it if you want....